src/actions/nodes.js

```diff
@@ -75,11 +75,16 @@ export function addNodeMetadata(data) {
  */
 export function loadNodeData(nodeID) {
   return async function(dispatch, getState) {
-    const { asyncDataSource, node } = getState();
+    const { asyncDataSource, node, flags } = getState();
 
     dispatch(toggleNodeClicked(nodeID));
 
-    if (asyncDataSource && nodeID && !node.fetched[nodeID]) {
+    if (
+      asyncDataSource &&
+      nodeID &&
+      !node.fetched[nodeID] &&
+      flags.meta === true
```

Comment:
Could this be changed to just `flags.meta` (instead of `flags.meta === true`), or does it need to be this specific?

Reply:
sure, we can just use `flags.meta` :)